examples/pkg/index-master.txt

```diff
@@ -1 +1,10 @@
-reachc: error: ./index-master.rsh:5:27:import from: Cannot install packages; did you mean to run with `--install-pkgs`?
+reachc: error: RI0003
```

Comment:
I think you should work on this format more. I like the rust style:

```
error[E0001]: unreachable pattern
  --> src/main.rs:10:9
   |
10 |         Known => 1,
   |         ^^^^^ this is an unreachable pattern
   |
```

The URLs need an `https` for link discoverers to turn on.